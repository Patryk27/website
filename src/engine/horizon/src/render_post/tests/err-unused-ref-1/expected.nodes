Element(
    Spanned {
        inner: Element {
            name: Spanned {
                inner: "ref",
                span: Span {
                    start: 1,
                    end: 4,
                },
            },
            attrs: [
                Attr {
                    name: Spanned {
                        inner: "id",
                        span: Span {
                            start: 5,
                            end: 7,
                        },
                    },
                    value: Some(
                        Spanned {
                            inner: "a",
                            span: Span {
                                start: 8,
                                end: 11,
                            },
                        },
                    ),
                },
            ],
            children: [
                Text(
                    Spanned {
                        inner: "\n    http://a.com\n",
                        span: Span {
                            start: 12,
                            end: 30,
                        },
                    },
                ),
            ],
        },
        span: Span {
            start: 0,
            end: 36,
        },
    },
)
Text(
    Spanned {
        inner: "\n\n",
        span: Span {
            start: 36,
            end: 38,
        },
    },
)
Element(
    Spanned {
        inner: Element {
            name: Spanned {
                inner: "ref",
                span: Span {
                    start: 39,
                    end: 42,
                },
            },
            attrs: [
                Attr {
                    name: Spanned {
                        inner: "id",
                        span: Span {
                            start: 43,
                            end: 45,
                        },
                    },
                    value: Some(
                        Spanned {
                            inner: "b",
                            span: Span {
                                start: 46,
                                end: 49,
                            },
                        },
                    ),
                },
            ],
            children: [
                Text(
                    Spanned {
                        inner: "\n    http://b.com\n",
                        span: Span {
                            start: 50,
                            end: 68,
                        },
                    },
                ),
            ],
        },
        span: Span {
            start: 38,
            end: 74,
        },
    },
)
Text(
    Spanned {
        inner: "\n\n",
        span: Span {
            start: 74,
            end: 76,
        },
    },
)
Element(
    Spanned {
        inner: Element {
            name: Spanned {
                inner: "ref",
                span: Span {
                    start: 77,
                    end: 80,
                },
            },
            attrs: [
                Attr {
                    name: Spanned {
                        inner: "id",
                        span: Span {
                            start: 81,
                            end: 83,
                        },
                    },
                    value: Some(
                        Spanned {
                            inner: "c",
                            span: Span {
                                start: 84,
                                end: 87,
                            },
                        },
                    ),
                },
            ],
            children: [
                Text(
                    Spanned {
                        inner: "\n    http://b.com\n",
                        span: Span {
                            start: 88,
                            end: 106,
                        },
                    },
                ),
            ],
        },
        span: Span {
            start: 76,
            end: 112,
        },
    },
)
Text(
    Spanned {
        inner: "\n\n",
        span: Span {
            start: 112,
            end: 114,
        },
    },
)
Element(
    Spanned {
        inner: Element {
            name: Spanned {
                inner: "a",
                span: Span {
                    start: 115,
                    end: 116,
                },
            },
            attrs: [
                Attr {
                    name: Spanned {
                        inner: "ref",
                        span: Span {
                            start: 117,
                            end: 120,
                        },
                    },
                    value: Some(
                        Spanned {
                            inner: "a",
                            span: Span {
                                start: 121,
                                end: 124,
                            },
                        },
                    ),
                },
            ],
            children: [
                Text(
                    Spanned {
                        inner: "a.com",
                        span: Span {
                            start: 125,
                            end: 130,
                        },
                    },
                ),
            ],
        },
        span: Span {
            start: 114,
            end: 134,
        },
    },
)
Text(
    Spanned {
        inner: "\n",
        span: Span {
            start: 134,
            end: 135,
        },
    },
)
Element(
    Spanned {
        inner: Element {
            name: Spanned {
                inner: "a",
                span: Span {
                    start: 136,
                    end: 137,
                },
            },
            attrs: [
                Attr {
                    name: Spanned {
                        inner: "ref",
                        span: Span {
                            start: 138,
                            end: 141,
                        },
                    },
                    value: Some(
                        Spanned {
                            inner: "c",
                            span: Span {
                                start: 142,
                                end: 145,
                            },
                        },
                    ),
                },
            ],
            children: [
                Text(
                    Spanned {
                        inner: "c.com",
                        span: Span {
                            start: 146,
                            end: 151,
                        },
                    },
                ),
            ],
        },
        span: Span {
            start: 135,
            end: 155,
        },
    },
)
Text(
    Spanned {
        inner: "\n",
        span: Span {
            start: 155,
            end: 156,
        },
    },
)