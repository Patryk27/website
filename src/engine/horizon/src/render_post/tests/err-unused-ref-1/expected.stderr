[31merror:[0m unused ref: `b`
   â•­â”€[ <unknown>:5:9 ]
   â”‚
 5 â”‚ <ref id="b">
   â”‚         â”€â”¬â”€  
   â”‚          â•°â”€â”€â”€ unused ref: `b`
â”€â”€â”€â•¯

compilation failed
